R/verbatim_popup.R

```diff
@@ -6,7 +6,8 @@
 #'
 #' @param id (`character(1)`) the `shiny` id
 #' @param button_label (`character(1)`) the text printed on the button
-#' @param ... additional arguments to `[shiny::actionButton()]`
+#' @param type (`button` or `link`) should `[shiny::actionButton()]` or `[shiny::actionLink()]` be used.
```

Comment:
@param type (`character(1)`) specifying whether to use `[shiny::actionButton()]` or `[shiny::actionLink()]`

Reply:
fixed